### DISCUSSION TITLE
（萌新求教）最新Win10客户端

### DISCUSSION THREAD

**Discussion by Ab1iu** [c0re100/qBittorrent-Enhanced-Edition#627] [General]
我找了半天，全下载了找不到Win10用哪个，谢谢

**Comment by fgr178707** [comment]
用这个 
https://github.com/c0re100/qBittorrent-Enhanced-Edition/releases/download/release-4.6.7.10/qbittorrent_enhanced_4.6.7.10_qt6_x64_setup.exe

提示更新，暂时不要更新 V5.0.x 内存溢出了软件崩溃 详情: #617
